summary_workflows/differential_usage/specification/D_benchmark_specification.md

```diff
@@ -0,0 +1,133 @@
+# D2 benchmark specification
+
+## Synopsis
+
+Benchmark to test sensitivity and False Discovery Rate (FDR) of RNAseq-based identification of genes with differentially used poly(A) sites compared with ground truth (genes identified from orthogonal 3'end seq data).
+
+The metrics are computed for two types of datasets:
+
+- RNA-Seq data compared with 3'end sequencing data
+- Simulated RNA-Seq data compared with dataset used for simulation
+
+### Input data:
+
+Comparison of differential PAS usage predicted from RNA-Seq data with 3'end sequencing data
+
+1. List of genes with information on differentially used poly(A) sites identified from RNA-Seq data using the benchmarked tool
+2. List of genes with information on differentially used poly(A) sites identified from orthogonal 3'end seq dataset
+
+Comparison of differential PAS usage predicted from simulated RNA-Seq data with dataset used for simulation
+
+1. List of genes with information on differentially used poly(A) sites identified from simulated RNA-Seq data using the benchmarked tool
+2. List of genes with information on differentially used poly(A) sites identified from the dataset used for simulation
+
+### Metrics
+
+Based on the input data the following metrics are computed:
+
+1. Sensitivity (TPR, True Positive Rate) = TP/(TP+FN)
+2. FPR (False Positive Rate) = FP/(FP+TN)
+
+TP - true positives - genes identified by the tool and present in the orthogonal dataset  
+FP - false positives - genes identified by the tool and not present in the orthogonal dataset  
+FN - false negatives - genes not identified by the tool but present in the orthogonal dataset
```

Comment:
Thank you for the clear specification! 
TN missing though.

Also, I guess you mean with "genes identified", that these genes are considered significant in the sense of differential PAS usage between two conditions? So TP would be the number of genes with FDR < ALPHA (normally 0.05) identified by the participant and orthogonal ground truth dataset. 
But how do we identify differential PAS usage from ground truth?

Reply:
Again, I think TN can be determined if the ground truth is really what the name implies.

Reply:
From the same discussion today. A TN for the differential usage has a different meaning compared to the one in the identification benchmark.

Reply:
TN can be calculated without problems for differential usage and  FPR metric uses TN. I'll add TN to the list.

Reply:
Yes, for differential polyadenylation, TN is more informative than for identification of PAS.